jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gke.yaml

```diff
@@ -162,6 +162,7 @@
                 export GKE_CREATE_FLAGS="--max-nodes-per-pool=1000"
                 export CLOUDSDK_CONTAINER_USE_CLIENT_CERTIFICATE=True
                 export CLOUDSDK_API_ENDPOINT_OVERRIDES_CONTAINER="https://staging-container.sandbox.googleapis.com/"
+                export KUBE_GKE_IMAGE_TYPE="gci"
```

Comment:
This is a GKE test, `export KUBE_GKE_IMAGE_TYPE="gci"` is correct. The reason to just move these to gci rather than cloning is that these tests require enormous quota, and it is not feasible to get quota for a clone at this time.